src/utils/validationUtil.ts

```diff
@@ -304,10 +305,20 @@ export class ValidationUtil {
     return missingMandatoryFields;
   }
 
-  private static getValidationSchema(vehicleType: string, testStatus: string) {
+  private static getValidationSchema(
+    vehicleType: string,
+    testStatus: string,
+    ivaDefects?: IVADefect[] | null,
+  ) {
     if (!(vehicleType && testStatus)) {
       return null;
     }
+    if (ivaDefects && ivaDefects.length > 0) {
```

Comment:
You may need to test this, but rather than hard code the logic here for the ivaDefects, would it be possible to just add it to the TestResultsSchemaCarSubmitted schema that already exists.

(Please see comments on TestResultsSchemaCarSubmitted for more info)